packs/equipment/soaring-wings-major.json

```diff
@@ -34,7 +34,28 @@
             "title": "Pathfinder Treasure Vault (Remastered)"
         },
         "quantity": 1,
-        "rules": [],
+        "rules": [
+            {
+                "key": "RollOption",
+                "option": "soaring-wings-major",
+                "toggleable": true
+            },
+            {
+                "key": "BaseSpeed",
+                "predicate": [
+                    "soaring-wings-major"
+                ],
+                "selector": "fly",
+                "value": "max(20,@actor.system.movement.speeds.land.value)"
+            },
+            {
+                "key": "TokenEffectIcon",
+                "predicate": [
+                    "soaring-wings-major"
+                ],
+                "value": "icons/creatures/abilities/wings-birdlike-blue.webp"
+            }
+        ],
```

Comment:
Kind of feel like we should just give a fly speed. No real need for the toggle for something that in practice will always be on.